src/ui/map/styles.js

```diff
@@ -0,0 +1,48 @@
+module.exports = [
```

Comment:
Great idea to pull this out into its own concern, though I think we should discourage additional default styles; we're approaching the maximum number for the UI (and we're over it for the mobile UI)

Reply:
+1